### ISSUE TITLE
Setup Office Printer

### ISSUE THREAD

**Issue by nelsonic** [dwyl/hq#569]
The Epson Ecotank ET-2600 is currently collecting dust in the office. Can we revive it?

I need to print some pages and would prefer to setup a _system_ to streamline this in future. 🤞

**Comment by iteles** [comment]
Worked for me about a month ago but I haven't needed it since 🤷‍♀️ 

Pretty sure we just need to try to download the drivers correctly again... 🤞 

They're here: https://epson.com/Support/Printers/All-In-Ones/ET-Series/Epson-ET-2600/s/SPT_C11CF46201

Another note: this doesn't work for me through our current _Lenovo USB C Hub_, I have to use a [USB C to USB adapter](https://amzn.to/3AxOMKl) directly but it works flawlessly that way 👍

**Comment by nelsonic** [comment]
I printed a few pages this morning from my “old” Mac with USB A port. It worked fine. 👌🏻
thanks for the tip about the Lenovo USB hub not working for USB C. 🤦🏻
Maybe we should test it with the Apple Original USB C hub. 💭

**Comment by nelsonic** [comment]
@Stephanymtr please follow the link above shared by @iteles to (re)download the relevant printer driver + tools for the Epson Printer. And use the Apple Original USB C “dongle” (hub):

![E0772ED1-A40F-4D04-9D41-E0EAAFC8DE7F](https://user-images.githubusercontent.com/194400/188063650-a8b7d547-a9c8-4f36-b900-9ec380161600.jpeg)

test printing something for your course. 👌🏻
report back with your progress. Thanks! 🙏 

the printer is under the hot desk:
![82B6130B-2737-40CE-985F-9332D44279AD](https://user-images.githubusercontent.com/194400/188063953-bfdec5ca-2578-4ca9-8eb9-cb12e5243232.jpeg)

And the “dongle” is already plugged into the USB cable which is on the desk:

![02BFB309-6B70-4906-AFF3-A5AC94C07DC3](https://user-images.githubusercontent.com/194400/188064056-1878d25d-5e29-428d-bb81-8d460f4428d4.jpeg)

you will need to turn on the power at the extension on top of the desk:

![39048230-FB8D-472F-967D-6BBCFC4FD94F](https://user-images.githubusercontent.com/194400/188064360-fc9756e1-83ed-473b-acd5-bfeaa144f941.jpeg)

Then the power (on/off) switch on the printer:

![70FC59A5-3B1C-480E-BDDE-60031EBA7B73](https://user-images.githubusercontent.com/194400/188064539-a98b1f3a-8fca-45ec-8eed-61eed6e4505a.jpeg)

if you get stuck in setting this up on your MacBook Air, please LMK! (Comment to this issue!) 🙏

**Comment by Stephanymtr** [comment]
Quick update: It has been a week,  but I just now had some time to work on this issue. I downloaded the drivers, but I couldn't find the "dongle".  Someone must have moved it since it has already been a couple days.

**Comment by nelsonic** [comment]
@Stephanymtr thanks for taking a look. I put the USB C adapter (“dongle”) on the hot desk in the studio. Let us know if you see it. 🤞

**Comment by nelsonic** [comment]
@Stephanymtr were you able to get the printer working on your Laptop? 🖨️ 🤞

**Comment by nelsonic** [comment]
If we can get our _current_ **`Epson ET-2600`** to print photos e.g: https://github.com/bmhco/shared/issues/4 it would be awesome! 🤞 

<hr />

If for any reason it doesn't work, or quality isn't great, even after doing a fe print-head cleaning cycles ...
we can _consider_ **`Epson ET-8550`**
https://www.fnac.pt/Impressora-Fotografica-Epson-EcoTank-A3-ET-8550-Preto-Impressora-Impressora-Jacto-de-Tinta/a8614538
![image](https://user-images.githubusercontent.com/194400/203804140-6366f465-1257-453f-b57e-c665b04ea505.png)
Yes, it's "expensive" but it's _muuuch_ better, does **`A3`** and has a screen which is more user-friendly. 
[![image](https://user-images.githubusercontent.com/194400/203804731-5a397034-b2fc-4232-8c1d-3c052b0b6e72.png)](https://youtu.be/-x7JrB3kRJA)
https://youtu.be/-x7JrB3kRJA

Long-term review: https://youtu.be/GlWWwEIrz3A
[![image](https://user-images.githubusercontent.com/194400/203805254-c110ab1f-6c24-4a55-929c-1de2c58741d9.png)](https://youtu.be/GlWWwEIrz3A)

borderless A3 photo print: https://youtu.be/rH-flHMyFiI
[![image](https://user-images.githubusercontent.com/194400/203805794-2edd9576-77e7-4e60-9ede-2074f2813c97.png)](https://youtu.be/rH-flHMyFiI)

**Comment by iteles** [comment]
We have a Canon printer as well which seems like the right vehicle for photos, albeit not an ecotank printer.
Can we dig it up and see if it still works?

**Comment by nelsonic** [comment]
@iteles, yes we can _retrieve_ the [**`Canon PIXMA iP8750`**](https://github.com/dwyl/hq/issues/1#issuecomment-141631126) Printer from Gerês, next time we go there ... ⏳

**Comment by Stephanymtr** [comment]
Unsuccessful :(

I tried to make it work again. I checked everything: 
- printer was turned on
- software installed correctly.
- USB cable connected securely at both ends
- ran a printer check 
- printed test page out (through Mac)🥇 
- checked the ink containers, nothing clogged and closed the cover correctly
- printer wasn't titled or at an angle

However when I tried printing a photo online, or test page through EPSON or document from google docs printing noise occurs, paper goes through, but nothing prints.

I'd like to try it again using a different computer...maybe tomorrow ?

**Comment by nelsonic** [comment]
OK, I think the "Next Action" on this is to test it from a Windows PC. 🪟 
Specifically the Lenovo X1 Thinkpad. 💻 
Let's do this tomorrow at "Nap Time". 🛌

**Comment by nelsonic** [comment]
Connected the printer to the Lenovo ("CAD") PC as it's permanently on and connected to the DMZ Wifi network. 🖥️ 
@Stephanymtr testing printing now. 🖨️ 🤞

**Comment by Stephanymtr** [comment]
It works flawlessly. Thank you for your help. I printed all the images we need to get the playroom mood board started. I used the office paper (80g.m) as the printer didn't accept the 250g.m hard cover paper. 

![image](https://user-images.githubusercontent.com/91985721/205287866-3c24b614-f359-4cad-a7fc-6cd19cca3b28.jpeg)

**Comment by nelsonic** [comment]
Thanks for confirming it's working. 👍

**Comment by iteles** [comment]
@nelsonic As you've seen, the printing is now stripey and no amount of print utility nozzle cleaning has resolved the issue. 

Did I hear you say there was a tool required?

**Comment by nelsonic** [comment]
@iteles the printer needs a new home that is dust-free. Was thinking of putting up a shelf somewhere but don't know _where_. Once we figure that out, we can do a thorough clean of the nozzles and then everything should work tickety-boo. From my 2 mins of research it did not require any special tools (beyond what we already have in the house...).

**Comment by iteles** [comment]
I need to use it to print off shipping labels so I'd like to do this asap.

I suggest that it's fine under my desk ontop of the intray given how little it is used normally.

**Comment by nelsonic** [comment]
@iteles given that we now have a _much_ nicer printer 🖨 https://github.com/dwyl/studio/issues/60 
Where would you like it to "live" @home to be "safe" and out of reach of children who can (_will_) break it. 💔 💭

**Comment by iteles** [comment]
I set it up on the `home` network for now.

In the short term it's particularly useful in the upstairs coworking space where it is now because I can sit at the desk whilst kids are around and use it. But if it needs to be 'safe' then the only place it can be is the leisure room I reckon.

The second the coworking space starts to be used by the public it will need to move to allow more space.

**Comment by nelsonic** [comment]
Yeah, "the _only_ place" ... 😕 
I was thinking more like the **Basement Office** where we can wall-mount it and keep a dust cover on it. 💭 

My bad for asking a _toooo_ [open question](https://en.wikipedia.org/wiki/Open-ended_question) ... 🤦‍♂
